internal/cli/config.go

```diff
@@ -187,6 +187,8 @@ const (
 	outputBeginComment = "<!-- BEGIN_TF_DOCS -->"
 	outputContent      = "{{ .Content }}"
 	outputEndComment   = "<!-- END_TF_DOCS -->"
+
+	outputCheck = false
```

Comment:
This too should be removed. Make sure to run `make lint` locally, or check the [output](https://github.com/terraform-docs/terraform-docs/pull/458/checks?check_run_id=2504783389) of GitHub Action.